docs/fleet/fleet.md

```diff
@@ -7,7 +7,7 @@ The Fleet section of Fleet and Discovery gives you an overview of all the Semate
 
 At first glance, you can see the following data in the table:
 
-- Host: Names of the hosts where the Sematext Agent is installed. You can either click on the name to open the Agent Information Panel or click on the expand view to reveal the list of Discovered Services available on the specific host. For further details, refer to the individual sections below in the document.
+- Host: Names of hosts where the Sematext Agent is installed and running. You can either click on the name to open the Agent Information Panel or click on the expand view to reveal the list of Discovered Services available on the specific host. For further details, refer to the individual sections below in the document.
```

Comment:
Agent Information Panel looks like the name of something one should see in the UI.  Maybe this is referring to some Info tab?
list of Discovered Services ==> list of discovered services
available on the specific host ==> running on the specific host
individual sections below in the document ==> individual sections below

Reply:
> Agent Information Panel 
It refers to the full flyout shown when one clicks a hostname in Fleet